site/source/docs/getting_started/FAQ.rst

```diff
@@ -140,7 +140,8 @@ That error can happen when loading the page using a ``file://`` URL, which works
 in some browsers but not in others. Instead, it's best
 to use a local webserver. For example, Python has one built in,
 ``python -m http.server`` in Python 3 or ``python -m SimpleHTTPServer``
-in Python 2. After doing that, you can visit ``http://localhost:8000/``.
+in Python 2. After doing that, you can visit ``http://localhost:8000/``. You can
```

Comment:
I suppose while we are editing this line, we can remove the Python2 bit, since emscripten now requires Py3.

Reply:
Yeah, it's hard to even install python2 on macs anymore, and IIUC Linux distros default to py3 as well. But leaving it here isn't really harmful either, so, either way.

Reply:
Hmm, let's leave it, seems like it might help some people even if rarely.